### PR TITLE
Improve waitForBuildProcess

### PR DESCRIPTION
**Pull request by stefanhaller**
- Improve the default pattern for build processes.
- Allow specifying a search pattern instead of true (let me know what you think!)

See #328.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -345,8 +345,11 @@
                 "default": false
               },
               "waitForBuildProcess": {
-                "markdownDescription": "(experimental) Prevents the extension of auto-reloading. With this linking failure might can be avoided.",
-                "type": "boolean",
+                "markdownDescription": "(experimental) Prevents the extension of auto-reloading. With this linking failure might can be avoided. Can be true to use a default pattern that works for most cases, or a string to pass your own search pattern for processes.",
```
Comment (matepek) [inline package.json:348]: Pls mention that the search pattern is regex.

**Review by matepek — CHANGES_REQUESTED** [review]
If you want a release, pls modify changelog too

File: package.json (hunk 1 above)
Comment (stefanhaller) [reply package.json]: Is this enough? 280d5a8046f1

Comment (matepek) [reply package.json]: yes, thanks

**Review by matepek — DISMISSED** [review]

**Pull request merged**